Snowdog_Components/components/Templates/product-view/product-view--ee.hbs

```diff
@@ -1,7 +1,7 @@
-<div class="page-wrapper">
+<div class="page-wrapper catalog-product-view">
     {{ render '@header' }}
 
-    <main class="product-view container">
+    <main class="product-view container" id="maincontent">
```

Comment:
please stick to this formatting rule:
if an element has more than 1 attribute, then each attribute should be in a separate line like this:
```suggestion
    <main 
        class="product-view container" 
        id="maincontent"
   >
```
please adjust in all your changes